compiler/compiler.go

```
@@ -159,3 +178,13 @@ func (c *FluxCompiler) addConstant(val interface{}) int {
 	c.chunk.Constants = append(c.chunk.Constants, val)
 	return len(c.chunk.Constants) - 1
 }
+
+func (c *FluxCompiler) compileBlock(block *ast.BlockExpr) error {
```

golangci-lint (1.64.8)

func `(*FluxCompiler).compileBlock` is unused

(unused)